processing/fb_event_processor.coffee

```diff
@@ -54,9 +55,10 @@ exports.extractAndSendEvents = (feed) ->
             location: getLocationInfo FBevent
             isAllDay: FBevent.end_time?
             startTime: startTime
-            endTime: if FBevent.end_time then new Date(FBevent.end_time).getTime() else null
+            endTime: if FBevent.end_time then new Date(FBevent.end_time).getTime() else 0
```

Comment:
I changed it from null to 0 since we are now checking to see if `start_time > (new Date).getTime() or end_time > (New Date).getTime()` when giving upcoming events to the client.

Do you think this is the best approach?